Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -6001,6 +6010,16 @@ class ConstructPoint_3 {
   */
   Kernel::Point_3 operator()(const CGAL::Origin &CGAL::ORIGIN);
 
+  /*!
+    returns `p`.
+
+    \note It is advised to return a const reference to `p` to avoid useless copies.
+
+    \note This peculiar requirement is necessary because some CGAL structures such as triangulations
+    internally manipulate points whose type might `Point_3` or `Weighted_point_3`.
```

Comment:
```suggestion
    \note This peculiar requirement is necessary because some \cgal structures such as triangulations
    internally manipulate points whose type might be `Point_3` or `Weighted_point_3`.
```